installer/volcano-development.yaml

```diff
@@ -8527,7 +8527,7 @@ rules:
     verbs: ["create", "get", "list", "watch", "delete"]
   - apiGroups: ["batch.volcano.sh"]
     resources: ["jobs"]
-    verbs: ["get", "list", "watch", "update", "delete"]
+    verbs: ["create", "get", "list", "watch", "update", "delete"]
```

Comment:
Perhaps both helm chart and yaml file should been modified.

Reply:
Besides, maybe this fix should also been included in the latest three releases.

Reply:
The pr will be merged to release-1.8 as well.  The release-1.8 is the first release branch that includes jobflow.

Reply:
> The pr will be merged to release-1.8 as well. The release-1.8 is the first release branch that includes jobflow.

OK, got it.